Reword admonitions for switching SCC accounts (#3514)
theforeman/foreman-documentation: guides/common/modules/proc_switching-scc-accounts.adoc

[WARNING]
====

> The SCC Manager plugin does not support multiple SCC accounts within one organization but for the purpose of switching SCC accounts.

Reviewer: ```suggestion
The SCC Manager plugin does not support multiple SCC accounts within one organization, except for when switching SCC accounts.
```
I'm not entirely sure about this, but it would be nice to replace 'but' with something that's more in line with simple English.

Author: I like this suggestion -> applied.
